### PR TITLE
DOT-1352 microtasking

### PR DESCRIPTION
**Pull request by edwh**
Add Workbench to navbar, using placeholder icon for now.

Various CSS fettling to make the Talk and Restarters navbars match up.

Show the navbar in microtasking when logged out.

Rename microtasking to 'workbench' externally.

### CODE REVIEW

**Review by ngm — COMMENTED** [review]
Looks good to me!  Just a question as to bootstrap.scss needed all the imports adding.  

Other than waiting on the new icons, ready to merge.

File: resources/global/css/bootstrap.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import "~bootstrap/scss/functions";
```
[the remark this answers is not in the document]
Comment (edwh) [reply resources/global/css/bootstrap.scss:1]: This is an odd one - it shows as a diff on here, but the files are showing as identical in the branch.

I don't understand that.  I think you should probably just merge it and then check the file afterwards to make sure it hasn't really changed.

[the remark this answers is not in the document]
Comment (ngm) [reply resources/global/css/bootstrap.scss:1]: Ah OK, probably just whitespace or special characters or similar. :+1:

**Comment by edwh** [comment]
Good question - looks like a merge issue or something my IDE did.  I'll 
have a look and see if they're needed.

------ Original Message ------

**Pull request merged**